app/tests/unit/forms/email/emailService.spec.js

```diff
@@ -253,8 +249,6 @@ describe('public methods', () => {
     expect(result).toEqual('ret');
     expect(formService.readForm).toHaveBeenCalledTimes(1);
     expect(formService.readForm).toHaveBeenCalledWith(form.id);
-    expect(formService.readSubmission).toHaveBeenCalledTimes(1);
-    expect(formService.readSubmission).toHaveBeenCalledWith(currentStatus.formSubmissionId);
```

Comment:
i guess because we are mocking `formService.readSubmission` it doesnt think it's worth testing (?)
but these tests do seem to have some value